core/src/main/java/org/ethereum/beacon/core/BeaconBlockBody.java

```diff
@@ -12,8 +14,12 @@
 import org.ethereum.beacon.core.operations.ProofOfCustodyResponse;
 import org.ethereum.beacon.core.operations.ProofOfCustodySeedChange;
 import org.ethereum.beacon.core.operations.ProposerSlashing;
+import org.ethereum.beacon.util.ssz.annotation.SSZSerializable;
+import org.ethereum.beacon.util.ssz.annotation.SSZTransient;
 
+@SSZSerializable
 public class BeaconBlockBody {
+  @SSZTransient
```

Comment:
Looks strange that you will need to annotate static fields. IMHO statics should be ignored by serializer

Reply:
fixed